src/h3/mod.rs

```diff
@@ -891,6 +905,8 @@ impl Connection {
         let mut d = [0; 8];
         let mut b = octets::OctetsMut::with_slice(&mut d);
 
+        conn.stream_priority(stream_id, 0, true)?;
```

Comment:
`urgency` need to be a default value, not 0?

Reply:
Thanks for commenting on this, I realise my intention was not obvious. quiche's priority space is 0-255, while HTTP `urgency` is 0-7. In this PR it's mapped 1:1, but I want to boost unidirectional control and qpack streams priority above all else. So I picked the lowest value allowed, 0, (We'll ignore push streams for now because quiche does not support them yet). By picking 0, this means there is an overlap and these streams could get interfered with by requests on lower stream IDs and urgency=0.

One solution I was contemplating was to not map the spaces 1:1 but to right shift urgency so that there was space in the low end (along with all the spare space in the top end). I had something like that in the old branch but think it got lost in this one. So I'll add a followup commit that does this and adds some comments to explain things better.